boards/nxp/mimxrt595_evk/mimxrt595_evk_mimxrt595s_cm33.dts

```diff
@@ -407,17 +407,20 @@ zephyr_udc0: &usbhs {
 				label = "mcuboot";
 				reg = <0x00000000 DT_SIZE_K(128)>;
 			};
+			/* The MCUBoot swap-move algorithm uses the last 98 sectors
+			 * of the primary slot0 for swap status and move.
+			 */
 			slot0_partition: partition@20000 {
 				label = "image-0";
-				reg = <0x00020000 DT_SIZE_K(3076)>;
+				reg = <0x00020000 (DT_SIZE_M(3) + DT_SIZE_K(98 * 4))>;
 			};
-			slot1_partition: partition@321000 {
+			slot1_partition: partition@322000 {
 				label = "image-1";
-				reg = <0x00321000 DT_SIZE_K(3072)>;
+				reg = <0x00382000 DT_SIZE_M(3)>;
 			};
-			storage_partition: partition@621000 {
+			storage_partition: partition@622000 {
 				label = "storage";
-				reg = <0x00621000 DT_SIZE_M(57)>;
+				reg = <0x00682000 (DT_SIZE_M(58) - DT_SIZE_K(520))>;
```

Comment:
Why this change? Trying to understand the -520K from the base size?

Reply:
It is based on: 64M /*Flash size*/ - 6M /*slot0&1*/ - (128K+98*4K) /* MCUBoot, swap status& move*/